onnxruntime/test/providers/cpu/tensor/expand_test.cc

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Did you change or add any tests or is this copied as-is from the other file?

Reply:
It is copied from other file with added cases.